Update integrations.rst

zorianix opened the pull request:
Distinguish the difference between the regular and detailed versions of the SonarQube scanner. A brief tutorial was also added to configure the SonarQube API.

zorianix committed:
Update integrations.rst

Maffooch commented on docs/integrations.rst:
```diff
@@ -238,12 +238,31 @@ Snyk
 
-SonarQube
+SonarQube Scan (Aggregates findings per cwe, title, description, file_path.)
 ---------
 SonarQube output file can be imported in HTML format.
 
 To generate the report, see https://github.com/soprasteria/sonar-report
 
+Version: >= 1.1.0
+
+SonarQube Scan Detailed (mport all findings from sonarqube html report.)
+---------
+SonarQube output file can be imported in HTML format.
+
+To generate the report, see https://github.com/soprasteria/sonar-report
+
+Version: >= 1.1.0
+
+SonarQube API Import
+---------
```
Make the hyphens extend to the end of the line of text above them. It's a syntax error otherwise.

I've not used the.  SonarQube API, so not sure on the content.

zorianix replied:
I have verified Sonarqube API and have been using this currently and thats the reason i decided to update the document

zorianix replied:
Hypens have been updated now

Maffooch commented:
Also add _something_ to the description of this PR please.

zorianix committed:
Update integrations.rst

Update the documentation to include latest information on how to implement or use sonarqube api import feature

Maffooch committed:
Formatting

Maffooch committed:
Add missing hyphen
